user-org-service/app/models/user/CreateUserRequest.java

```diff
@@ -0,0 +1,333 @@
+package models.user;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author anmolgupta
+ * @desc POJO class for User Create
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class CreateUserRequest implements Serializable {
+
+    private static final long serialVersionUID = -1L;
+
+    private String id;
+    private String avatar;
+    private String countryCode;
+    private String dob;
+    private String email;
+    private Boolean emailVerified;
+    private String firstName;
+    private String gender;
+    private List<String> grade;
+    private List<String> language;
+    private String lastName;
+    private String location;
+    private String phone;
+    private Boolean phoneVerified;
+    private String profileSummary;
+    private Map<String, String> profileVisibility;
```

Comment:
profileVisibility won't come in request , this is internal field used by system.